app/templates/people/person/positions/minister/edit.hbs

```diff
@@ -0,0 +1,167 @@
+                        />
+                        </:content>
+                    </Item>
+                    <Item @labelFor="end-date">
+                        <:label></:label>
+                        <:content>
+                        <AuControlCheckbox
+                            @checked={{this.isCurrentPosition}}
+                            @label="Huidige positie"
+                            @identifier="current-position"
+                            @onChange={{fn (mut this.isCurrentPosition)}}
+                        />
+                        </:content>
+                    </Item>
+                    </:right>
+                </Card.Columns>
+                </:card>
+            </EditCard>
+
+            <EditCard>
+                <:title>Financiering</:title>
+                <:card as |Card|>
+                <Card.Columns>
+                    <:left as |Item|>
+                    <Item @alignTop={{true}}>
+                        <:label>FOD financiering</:label>
+                        <:content>
+                        <ul class="au-c-list-inline">
+                            <li class="au-c-list-inline__item">
+                            <AuControlRadio
+                                @label="Ja"
+                                @name="financing"
+                                @value={{this.willReceiveFinancing}}
+                                checked={{this.willReceiveFinancing}}
+                                @onChange={{fn (mut this.willReceiveFinancing) true}}
+                                @identifier="receives-financing"
+                            />
+                            </li>
+                            <li class="au-c-list-inline__item">
+                            <AuControlRadio
+                                @label="Nee"
+                                @name="financing"
+                                @value={{this.willReceiveFinancing}}
+                                checked={{not this.willReceiveFinancing}}
+                                @onChange={{fn (mut this.willReceiveFinancing) false}}
+                                @identifier="receives-no-financing"
+                            />
+                            </li>
+                        </ul>
+                        </:content>
+                    </Item>
+                    </:left>
+                </Card.Columns>
+                </:card>
+            </EditCard>
+
+            {{! TODO: add "Voorwaarde" section once file uploads are supported }}
+
+            <div>
+                <AuButton
+                    @iconAlignment="left"
+                    @loading={{this.save.isRunning}}
+                    @disabled={{this.save.isRunning}}
+                    class="au-u-margin-right-small"
+                    type="submit"
+                >
+                    Opslaan
+                </AuButton>
+                <AuLink @route="people.person.positions.minister">
```

Comment:
This should be [styled as a button](https://github.com/lblod/frontend-contact-hub/pull/154/files#diff-fc99dcdfb5b02a9c536c332ea5f414ef9cbbf0fbd4e70b37640f83d6a86694c3R262-R264).